Core.ApplicationServices/Contract/Write/IItContractWriteService.cs

```diff
@@ -24,6 +24,7 @@ Result<ExternalReference, OperationError> UpdateExternalReference(Guid contractU
             DeleteExternalReference(Guid contractUuid, Guid externalReferenceUuid);
 
         Result<ItContract, OperationError> AddRole(Guid contractUuid, UserRolePair assignment);
+        Result<ItContract, OperationError> AddRoleRange(Guid contractUuid, IEnumerable<UserRolePair> assignments);
```

Comment:
The methods in contract/dpr/usage services could be extracted to a common service, but I don't think it's worth it in this case